mix.exs

```diff
@@ -37,10 +37,10 @@ defmodule ExPlasma.MixProject do
       {:credo, "~> 1.4", only: [:dev, :test], runtime: false},
       {:dialyxir, "~> 1.0", only: [:dev], runtime: false},
       {:ethereumex, "~> 0.6.0", only: [:test]},
-      {:ex_abi, "~> 0.4.0"},
+      {:ex_abi, git: "https://github.com/ayrat555/ex_abi-1", ref: "4584a2e90f8acb0a1710e196a1a627b4f373a33d"},
```

Comment:
Do you plan to publish this to hex.pm? I think you will need to if you want to publish ex_plasma there.

Reply:
yes, I'm waiting for my pr (https://github.com/poanetwork/ex_abi/pull/42) to be merged